src/metabase/search/appdb/index.clj

```diff
@@ -150,7 +150,7 @@
   ([table-name]
    (boolean
     (when (exists? table-name)
-      (update-metadata! {:active-table table-name})))))
+      (update-metadata! {:active-table table-name :pending-table nil})))))
```

Comment:
Small silly bug here, we'd keep both pointers going to the same table, which would mean repeating all our writes.